File: app/src/main/java/org/mifos/mobile/ui/views/BeneficiariesView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.models.beneficiary.Beneficiary;
import org.mifos.mobile.ui.views.base.MVPView;

import java.util.List;

/**
 * Created by dilpreet on 14/6/17.
 */

public interface BeneficiariesView extends MVPView {

    void showUserInterface();

    void showError(String msg);

    void showBeneficiaryList(List<Beneficiary> beneficiaryList);
}

File: app/src/main/java/org/mifos/mobile/ui/views/BeneficiariesView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.models.beneficiary.Beneficiary
import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 14/6/17.
 */
interface BeneficiariesView : MVPView {
    fun showUserInterface()
    fun showError(msg: String?)
    fun showBeneficiaryList(beneficiaryList: List<Beneficiary?>?)
}